applications: nrf5340_audio: Add CSIS into headsets (#9268)
nrfconnect/sdk-nrf: doc/nrf/software_maturity.rst

          Receiving unidirectional or transceiving bidirectional audio using CIG and CIS.

          To emulate play and pause, the available context type for media is added or removed. This enables and disables streaming, respectively.

>           CSIS (Coordinated Set Identification Service) is implemented on the server side.

Reviewer: ```suggestion
          Coordinated Set Identification Service (CSIS) is implemented on the server side.
```

>           CSIS (Coordinated Set Identification Service) is added which makes initiator easier to search and connect with acceptors.

Reviewer 2: ```suggestion
          CSIS (Coordinated Set Identification Service) is implemented on the server side.
```
